segmentation/include/pcl/segmentation/impl/sac_segmentation.hpp

```diff
@@ -258,6 +258,15 @@ pcl::SACSegmentation<PointT>::initSACModel (const int model_type)
       }
       break;
     }
+    case SACMODEL_CYLINDER:
```

Comment:
Is there no sacmodel_normal_cylinder/ sacmodel_normal_cone ?

Seems odd having them here, when the 3 others does have "normal" in their names, to use it with SacSegmenationDromNormals.

Reply:
> Is there no sacmodel_normal_cylinder/ sacmodel_normal_cone ?
> 
> Seems odd having them here, when the 3 others does have "normal" in their names, to use it with SacSegmenationFromNormals.

Got to look my self and there isn't. But I guess we just keep the current naming instead of adding two new , with "normal" in the name, to supplement the others, event though there currently is not implementation without normals?

Reply:
You are correct: `SACMODEL_CYLINDER` and `SACMODEL_CONE` use normals, even though they don't have "normal" in their names. There are no cylinder and cone models that don't use normals. `SACMODEL_NORMAL_PLANE`, `SACMODEL_NORMAL_PARALLEL_PLANE`, and `SACMODEL_NORMAL_SPHERE` have "normal" in their names because there are equivalent models that don't use normals.
I also don't see much benefit in adding the new names `SACMODEL_NORMAL_CYLINDER` and `SACMODEL_NORMAL_CONE` for the existing models. Actually, it might cause confusion.